Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -6266,8 +6266,7 @@ RTCEncodedStreamsQuirkEnabled:
     WebKitLegacy:
       default: false
     WebKit:
-      PLATFORM(MAC): true
-      default: false
+      default: WebKit::defaultRTCEncodedStreamsQuirkEnabled
```

Comment:
A simpler equivalent solution would be to change this from "internal" to "preview"